tests/rptest/scale_tests/many_partitions_test.py

```diff
@@ -847,6 +853,8 @@ def _test_many_partitions(self,
             topic_partitions_per_shard,
             'topic_memory_per_partition':
             mib_per_partition * 1024 * 1024,
+            'topic_partitions_memory_allocation_percent':
+            DEFAULT_PARTITIONS_MEMORY_ALLOCATION_PERCENT,
```

Comment:
Why do we set it explicitly? This is also the default, right?

Reply:
It is the default yes. But once we want to test 3000k partitions we need to bump it to 15%. The 10% just so works out that with the same absolute assigned memory we get double the partition count.